src/router_node.c

```diff
@@ -226,9 +227,22 @@ static void AMQP_rx_handler(void* context, qd_link_t *link, pn_delivery_t *pnd)
     //        send it.
     //
     msg = qd_message_receive(pnd);
+
     if (!msg)
         return;
 
+
+    char repr[qd_message_repr_len()];
+
+    if (conn->log_message) {
+        qd_log(qd_message_log_source(), QD_LOG_TRACE, "Received %s on link %s",
```

Comment:
I think the log would look better if the link id was at the beginning and the message repr was at the end.